On The Map/Model/Udacity API Requests/Requests/StudentLocationRequest.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  StudentLocationRequest.swift
+//  On The Map
+//
+//  Created by Michal Hus on 2/24/20.
+//  Copyright © 2020 Michal Hus. All rights reserved.
+//
+
+import Foundation
+
```

Comment:
looks like this guy got moved or refactored out so if this were production code you'd wanna remove this file

Reply:
Woot?! No idea how that happened. LOL